test/metadataSpec.js

```diff
@@ -26,6 +26,7 @@ describe('Metadata', function () {
           expect(res.body).to.deep.equal({
             currency_code: null,
             currency_symbol: null,
+            ilp_prefix: null,
```

Comment:
in configSpec https://github.com/interledger/five-bells-ledger/pull/309/files#diff-2ed4457106d98f881aab4dcbdccd6b8dR81 it tests setting the value